docs/tasks.rst

```diff
@@ -144,6 +144,8 @@ Options:
 * **namespace_inject**: If set, the namespace tokens in files and filenames are replaced with the namespace's prefix
 * **namespace_strip**: If set, all namespace prefixes for the namespace specified are stripped from files and filenames
 * **namespace_tokenize**: If set, all namespace prefixes for the namespace specified are replaced with tokens for use with namespace_inject
+* **check_only**: If True, performs a test deployment (validation) of components without saving the components in the target org
+* **test_level**: Specifies which tests are run as part of a deployment.
```

Comment:
I would consider adding supported values (as test level is an enum). The changed line would look like belo:\
`* **test_level**: Specifies which tests are run as part of a deployment (valid values: NoTestRun, RunLocalTests, RunAllTestsInOrg).`

I skipped `RunSpecifiedTests` intentionally because that option requires list of the tests to run.

Reply:
Applies to all below.